docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  db:
+    image: postgis/postgis:14-3.3-alpine
```

Comment:
I'd suggest to check if there is a looser pinning here; maybe postgis/postgis:14-alpine - unless you specifically depend on 14-3.3 ?

Reply:
Does not seem to be https://hub.docker.com/r/postgis/postgis :(